### PR TITLE
Models dict

### PR DESCRIPTION
**Pull request by PierreGtch**
adds `braindecode.models.util.models_dict`

### CODE REVIEW

**Comment by robintibor** [comment]
Can you explain a bit what this does? And maybe add such explanation in the comments. Looks quite magic to me at the moment haha :)

File: braindecode/models/util.py [hunk 1]
```diff
@@ -146,3 +149,13 @@ def aggregate_probas(logits, n_windows_stride=1):
     """
     log_probas = log_softmax(logits, axis=1)
     return _pad_shift_array(log_probas, stride=n_windows_stride).sum(axis=0).T
+
+
+models_dict = {}
+
+
```
Comment (bruAristimunha) [inline braindecode/models/util.py:156]:
```suggestion

# For the models inside the init model, go through all the models 
# check those have the EEGMixin class inherited. If they are, add them to the list.
```

**Review by bruAristimunha — APPROVED** [review]

**Comment by bruAristimunha** [comment]
I added a small comment explaining the filling logic @robintibor. It is a for that is executed when loading the library that will allow you to access the list of models and is updated dynamically.

**Comment by PierreGtch** [comment]
@robintibor Haha it’s the same as for the `dataset_list` in MOABB
@bruAristimunha Thanks!

**Comment by bruAristimunha** [comment]
Thank you Pierre!

**Pull request merged**